src/pages/home.css

```diff
@@ -0,0 +1,212 @@
+#page {
```

Comment:
I think this file should be moved right now I'm just putting everything in corresponding component folders. So this could go in src/components/home/styles.css